.github/workflows/codeQL-analysis.yaml

```diff
@@ -0,0 +1,24 @@
+---
+name: Static analysis with CodeQL
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  schedule:
+    - cron: '0 8 * * 1' # Monday - 8am - UTC
+
+jobs:
+  analyze:
+    name: Static analysis with CodeQL
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v2
+        with:
+          languages: python
```

Comment:
Isn't there any more language we may find interesting?

Reply:
I don't think there are any other languages in the 'runner-manager' case